empire/server/server.py

```diff
@@ -1133,10 +1129,14 @@ def get_agents_stale():
                         "lost_limit": agent.lost_limit,
                         "architecture": agent.architecture,
                         "proxy": agent.proxy,
-                    }
+                    },
+                    Session()
+                    .query(models.Agent)
+                    .filter(models.Agent.killed == False, models.Agent.stale == True)
```

Comment:
Unfortunately, filtering on stale within the db query is not possible because its a [hybrid property](https://github.com/BC-SECURITY/Empire/blob/master/empire/server/database/models.py#L110).